base_deploy_kosko_stage.yml

```diff
@@ -46,7 +46,12 @@ variables:
           echo '---' \; -exec cat {} \; \
         >> ${CI_PROJECT_DIR}/manifest.yaml
       fi
-    - kubectl apply -f ${CI_PROJECT_DIR}/manifest.yaml
+    - |
+      if [[ -z $PRODUCTION ]]; then
+        kubectl create ns ${KUBE_NAMESPACE} || true
+        kubectl annotate ns ${KUBE_NAMESPACE} azure-pg-admin-user=sample-next-app || true
+      fi
+    - kapp -y deploy -a ${CI_PROJECT_NAME} --namespace ${KUBE_NAMESPACE} -f ${CI_PROJECT_DIR}/manifest.yaml
```

Comment:
can we add a `$KAPP_DEPLOY_ARGS` at the end of the kapp command so one can add parameters ?

Reply:
Done.